primitives/src/v2/mod.rs

```diff
@@ -1403,6 +1403,19 @@ pub struct DisputeState<N = BlockNumber> {
 	pub concluded_at: Option<N>,
 }
 
+#[cfg(feature = "std")]
+impl MallocSizeOf for DisputeState {
+	fn size_of(&self, ops: &mut MallocSizeOfOps) -> usize {
+		// BitVec is converted to slice to get its size dynamically.
+		// It's equal to calling .capacity() * sizeof::<u8>() but without hardcoding the actual T
```

Comment:
I'd suggest using exactly `.capacity() * 8`. Not sure what `T` has to do with it, it's not used in `BitVec`.

Reply:
I wanted not to assume that the BitVec uses u8 by default - this is the T I'm referring to.
E.g. for some reason someone changes the `BitVec` declaration in `DisputeState` to something like this:
```
pub validators_for: BitVec<u32, bitvec::order::Lsb0>
```
Now the computation will be wrong, because we need `validators_for.capacity() * size_of<u32>`.

Reply:
It's a bitvec... of course the capacity is in bits :man_facepalming: